settings.gradle.kts

```diff
@@ -1,2 +1,3 @@
 include(":app")
-rootProject.name = "AndroidAcademyApp"
\ No newline at end of file
+rootProject.name = "AndroidAcademyApp"
```

Comment:
move rootProject.name = "AndroidAcademyApp" to top

Reply:
Agreed with separation for libs/plugins/modules
version catalog and TYPESAFE_PROJECT_ACCESSORS - sounds good too

Reply:
Done